docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,73 @@
+verified the migration process. CNIs and versions that are not listed here
+might also work. Please create an issue if you run into problems during the
+migration to Antrea. During the migration process, no Kubernetes resources
+should be created or deleted, otherwise the migration process might fail or
+some unexpected problems might occur.
+
+| CNI     | Version |
+|---------|---------|
+| Calico  | v3.26   |
+| Flannel | v0.22.0 |
+
+The migration process is divided into three steps:
+
+1. Clean up the old CNI.
+2. Install Antrea in the cluster.
+3. Restart all Pods in the cluster in-place.
+
+## Clean up the old CNI
+
+Before installing Antrea in the cluster, the old CNI needs to be cleaned up. The
+cleanup process is different for different CNIs, but the basic idea is to
+remove the old CNI DaemonSet, Deployment, and CRDs from the cluster. The most
+common way is to use the `kubectl delete` command to delete the old CNI, just
+like the following example:
+
+```bash
+$ kubectl delete -f <OLD_CNI_MANIFEST>
+```
+
+## Install Antrea
+
+The second step is to install Antrea in the cluster. You can follow the
+[installation guide](https://github.com/antrea-io/antrea/blob/main/docs/getting-started.md)
+to install Antrea. The following is an example of installing Antrea v1.14.1:
+
+```bash
+$ kubectl apply -f https://github.com/antrea-io/antrea/releases/download/v1.14.1/antrea.yml
+```
+
+## Restart Pods
+
+After Antrea is installed in the cluster, the next step is to restart all
+Pods in the cluster in-place by the following command. This step will create
+a DaemonSet in the cluster, which will restart all Pods in the cluster in-place.
+
+```bash
+$ kubectl apply -f https://github.com/antrea-io/antrea/releases/download/v1.15.0/antrea-migrator.yml
+```
+
+The reason for restarting all Pods is that Antrea needs to take over the
+network management and IPAM from the old CNI. In order to avoid the Pods
+being rescheduled, we restart all Pods in-place by deploying a DaemonSet
+named `antrea-migrator`, which will run a Pod on each Node. The
+`antrea-migrator` Pod will stop all containerd tasks of Pods with CNI network on
+each Node, and the containerd tasks will be restarted by the containerd
+service. In this way, all Pods in the cluster will not be rescheduled but will
+be restarted in-place.
+
+The restart result will be similar as follows, and every Pod's `RESTARTS` will
+be increased by 1:
+
+```bash
+$ kubectl get pod -o wide
+NAME                               READY   STATUS    RESTARTS      AGE    IP          NODE          NOMINATED NODE   READINESS GATES
+migrate-example-6d6b97f96b-29qbq   1/1     Running   1 (24s ago)   2m5s   10.10.1.3   test-worker   <none>           <none>
+migrate-example-6d6b97f96b-dqx2g   1/1     Running   1 (23s ago)   2m5s   10.10.1.6   test-worker   <none>           <none>
+migrate-example-6d6b97f96b-jpflg   1/1     Running   1 (23s ago)   2m5s   10.10.1.5   test-worker   <none>           <none>
+```
```

Comment:
Add a paragraph to say after which condition (like all Pods of antrea-migrator are ready), it's safe to remove the DaemonSet.